docs/usage.md

```diff
@@ -269,6 +269,30 @@ With this configuration in place deployment should happen automatically every ti
 
 There is also a [Shiny server application](https://posit.co/download/shiny-server/), which you can install on your own infrastruture and use to host applications yourself.
 
+## Gene set enrichment analysis
+
+Currently, two tools can be used to do gene set enrichment analysis.
+
+### g:Profiler
```

Comment:
Nope, the tool itself is called [g:Profiler](https://academic.oup.com/nar/article/47/W1/W191/5486750); I decided to add the name of the tool itself instead of the R package gprofiler2. Should I change that?

I went for an alphabetic ordering, but sure, will change!